File: src/main/java/Box.java
public class Box {
    float width;
    int height;
    float length;

    public Box() { }

    public boolean validate(float length, float width, int height) {
        if (this.length>= length && this.width >=width && this.height >=height) {
            return true;
        }
        return false;
    }
}

File: src/main/kotlin/Box.kt
package com.hanson1

open class Box {
    internal var width: Float = 0.toFloat()
    internal var height: Int = 0
    internal var length: Float = 0.toFloat()

    constructor() {}

    fun validate(length: Float, width: Float, height: Int): Boolean {
        return (this.length >= length && this.width >= width && this.height >= height)
    }
}

